### ISSUE TITLE
Passwords don't match - password reset

### ISSUE THREAD

**Issue by BradRyan** [meteor-useraccounts/core#188]
I've been struggling with a couple sign-in related issues.  Perhaps someone could point me in the right direction?

1) I'm getting an error when I use the password reset after following the "forgot password" link.  When I follow the link I receive in the email I go to the reset-password page and am able to type in my passwords.  When I finish typing in the "New Password (again)" field I get an error that my passwords don't match, but they definitely do.  The submit button disappears, but I am able to use the submit key and I am logged in.  

Seems like a bug in the 2nd account validation logic.  I never changed it, but I didn't see any comments about this in the github issues so my guess is I somehow managed to screw something up.  Yet, when I create a new account I don't see this password matching error.

Example password reset link:
http://localhost:3000/reset-password/UPC-pXGhh3AflPn7hBKgXlodM7mGxhVAXXG_LEjM7TJ

Screenshot of the failed validation at password-reset:
![screen shot 2014-12-03 at 10 52 26 pm](https://cloud.githubusercontent.com/assets/9065296/5294872/ae6a3168-7b41-11e4-8b64-4cbbadac1694.png)

Successful validation at account creation:
![screen shot 2014-12-03 at 11 20 17 pm](https://cloud.githubusercontent.com/assets/9065296/5295678/ee603f02-7b4f-11e4-859f-9940c43eebaa.png)

2) This is more of a nice to have... Any ideas how I could make the reset password form open in a modal on my home page?

**Comment by juliomac** [comment]
I think I could reproduce it. See issue: https://github.com/meteor-useraccounts/bootstrap/issues/39

I am also using modal. Could that have any weird effect?

**Comment by BradRyan** [comment]
Whoops... sorry I never investigated further since I decided not to require duplicate password entries.  I've since rolled my own for a new application.  Sorry to leave you hanging @splendido!

**Comment by juliomac** [comment]
Hi @BradRyan , I would be interested to learn how you solved. I had too take password_again out as I couldn't find a solution.

**Comment by splendido** [comment]
@juliomac do you have any public reproduction for the problem?

**Comment by splendido** [comment]
@ntoka, what about you: any public reproduction for this?

**Comment by ntoka** [comment]
@splendido No, sorry, I finally decided to switch to my own user creation solution.

**Comment by juliomac** [comment]
Hi @splendido , I have reverted to request one single password on my production site. However I have kept a copy with password verification on amazonaspalace.meteor.com.

Hope this helps!

**Comment by nicooprat** [comment]
Hope it helps someone else: I also got the error "passwords don't match" (while they did), because I didn't set the option `enablePasswordChange: true`.

I think the `atForm` template should show nothing when having the state `changePwd` set and this option set to `false` (default value).
